src/core/server/metrics/metrics_service.ts

```diff
@@ -71,6 +74,8 @@ export class MetricsService
   private async refreshMetrics() {
     this.logger.debug('Refreshing metrics');
```

Comment:
May I ask you to remove this log? It doesn't provide any value anymore - we now log the metrics.